pkg/api-server/handler.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2020 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiserver
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/timestamp"
+
+	"github.com/kubermatic/kubecarrier/pkg/apis/apiserver/v1alpha1"
+	"github.com/kubermatic/kubecarrier/pkg/internal/version"
+)
+
+type kubecarrierHandler struct{}
```

Comment:
Please call this KubeCarrierServer or VersionServer (ref. https://github.com/kubermatic/kubecarrier/pull/398/files#r413441230)

I think Handler is missleading as it's not a normal http handler, but the implementation of the `KubeCarrierServer` interface.